pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -1361,7 +1185,7 @@ func (p *PriorityQueue) nominatedPodToInfo(np PodRef) *framework.PodInfo {
 	pod := np.ToPod()
```

Comment:
the comment is outdated? Or it should point out that activeQ.lock() is locked too.

Reply:
Updated the comment